charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.36.0
+version: 9.37.0
```

Comment:
```suggestion
version: 9.36.1
```

Reply:
As we are resolving a bug not adding new functionality it should be a Patch version, not a Minor version